### DISCUSSION TITLE
Major Transmission PR

### DISCUSSION THREAD

**Discussion by robd003** [transmission/transmission#4778] [Show and tell]
Looks like a lot of people will be downloading Transmission 4 today:

https://www.macrumors.com/2023/02/09/bittorrent-transmission-4-apple-silicon-support/

Congrats again to everyone who has worked so hard these past two years, 4.0 rocks!

**Comment by ckerr** [comment]
> And after all these years, this app is still complete trash. qBittorrent is a hundred times better. I prefer the simple UI of Transmission, but everything else about it is terrible.

Thanks, Internet

**Reply by robd003** [reply]
I find it kind of funny because qBittorrent never worked while connected to my work VPN whereas Transmission has always been reliable.

Don't let the haters get you down @ckerr, everyone here thinks you're the man for the **years of great work** on this project.
